### PR TITLE
remove redundant codes and update the function for fs2 feature

### PR DESCRIPTION
**Pull request by ChenX17**
remove redundant codes and update the function for fs2 feature

### CODE REVIEW

**Review by RMSnow — APPROVED** [review]
LGTM

File: utils/mel.py [hunk 1]
```diff
@@ -231,9 +231,6 @@ def extract_mel_features_tts(
 
         spec = torch.matmul(mel_basis[str(cfg.fmax) + "_" + str(y.device)], spec)
         spec = spectral_normalize_torch(spec)
-        spec = spec.squeeze(0)
```
Comment (lmxue) [inline utils/mel.py:234]: Are you sure it works to delete the code in line 234, which is not a repeated code?

**Review by lmxue — COMMENTED** [review]
Please check out the comments.

**Review by zhizhengwu — CHANGES_REQUESTED** [review]
line 234 needs to resolve before moving forward

File: utils/mel.py (hunk 1 above)
Comment (ChenX17) [reply utils/mel.py:234]: Yes, it's same as lines 165 to 168.

**Review by lmxue — APPROVED** [review]

**Comment by zhizhengwu** [comment]
LGTM

**Review by zhizhengwu — APPROVED** [review]

**Pull request merged**